src/main/java/de/ellpeck/rockbottom/content/ContentManager.java

```diff
@@ -27,8 +31,9 @@
     public static void init(IGameInstance game) {
         new ConstructionRecipeLoader().register();
         new SmeltingRecipeLoader().register();
-        new StructureLoader().register();
-        new MortarRecipeLoader().register();
+        new SmithingRecipeLoader().register();
+		new MortarRecipeLoader().register();
+		new StructureLoader().register();
```

Comment:
a bit strange formatting

Reply:
blame github and git. its not how it actually looks like

Reply:
Problem is that these are tabs, normally it should be a lot of spaces instead of tabs